kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,14 @@
+package com.kodilla.stream.array;
+
+import java.util.stream.IntStream;
+
+public interface ArrayOperations {
+    public static double getAverage(int[] numbers) {
+        if (numbers == null || numbers.length == 0) {
+            throw new IllegalArgumentException();
+        }
+        return IntStream.range(0, numbers.length)
```

Comment:
zrobił Pan to zgodnie z poleceniem, ale nie rozumiem czemu w zadaniu sugerowano aby wykorzystać range.
Dużo prościej byłoby napisaś:
IntStream.of(numbers).average()

Reply:
Jak zawsze, dobrze się zapoznać z biblioteką stream, ale tego trochę jest.